src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -59,6 +67,18 @@ public ConfigureOptions(
         [Option("runtime.graphql.depth-limit", Required = false, HelpText = "Max allowed depth of the nested query. Allowed values: (0,2147483647] inclusive. Default is infinity. Use -1 to remove limit.")]
         public int? DepthLimit { get; }
 
+        [Option("runtime.graphql.enabled", Required = false, HelpText = "Configure enable/disable flag for graphQL runtime options.")]
+        public bool? RuntimeGraphQLEnabled { get; }
+
+        [Option("runtime.graphql.path", Required = false, HelpText = "Configure path for graphQL runtime options.")]
+        public string? RuntimeGraphQLPath { get; }
+
+        [Option("runtime.graphql.allow-introspection", Required = false, HelpText = "Configure allow-introspection for graphql runtime options.")]
+        public bool? RuntimeGraphQLAllowIntrospection { get; }
+
+        [Option("runtime.graphql.multiple-mutations.create.enabled", Required = false, HelpText = "Configure multiple-mutations.create.enabled in graphql runtime options.")]
+        public bool? RuntimeGraphQLMultipleMutationsCreateEnabled { get; }
```

Comment:
```suggestion
        [Option("runtime.graphql.enabled", Required = false, HelpText = "Enable DAB's GraphQL endpoint.")]
        public bool? RuntimeGraphQLEnabled { get; }

        [Option("runtime.graphql.path", Required = false, HelpText = "Customize DAB's GraphQL endpoint path.")]
        public string? RuntimeGraphQLPath { get; }

        [Option("runtime.graphql.allow-introspection", Required = false, HelpText = "Allow GraphQL introspection requests.")]
        public bool? RuntimeGraphQLAllowIntrospection { get; }

        [Option("runtime.graphql.multiple-mutations.create.enabled", Required = false, HelpText = "Enable multiple-mutation create operations on DAB's generated GraphQL schema.")]
        public bool? RuntimeGraphQLMultipleMutationsCreateEnabled { get; }
```

nit: usage of "GraphQL" and and added more specific helptext

Reply:
updated